.eslint/__tests__/svg-needs-viewBox.test.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @fileoverview All svgs should have a viewBox attribute
+ * @author Glorilí Alejandro
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../rules/svg-needs-viewBox.js');
+
+const RuleTester = require('eslint').RuleTester;
```

Comment:
question (non-blocking): why did you go with the `require` instead of the `import` statement you have below (commented out)?

Reply:
It won't work with `import`; I believe it's because we're at the root? I talked about it w/ @zahnster but can't remember the exact reason. Any insight here @zahnster?